deployment/ansible/roles/raster-foundry.jenkins/handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Restart Jenkins
+  service: name=jenkins state=restarted
+
+- name: Restart Nginx
```

Comment:
This handler is already defined in the `azavea.nginx` role, so I don't think it needs to be redefined here.